Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -752,6 +948,26 @@ public override void Update(float elapsedClockSeconds)
                             AuxResPressurePSI += dp * EmergAuxVolumeRatio;
                         }
                     }
+                    else if ((Car as MSTSWagon).SupplyReservoirPresent && BrakeLine1PressurePSI < 15.0f)
+                    {
+                        // Supply res air directed to BC feed line to ensure full emergency force on cars with no emergency res
+                        // Only activated with brake pipe pressure lower than 10-18 psi (assuming 15 is the design point)
+
+                        // Ratio of displacement (dummy brake cylinder) volume to supply res volume
+                        // We are supplying pressure to the brake cylinder line, not the brake cylinder itself
+                        float dispalcementSupplyVolumeRatio = AuxResVolumeM3 / AuxCylVolumeRatio / SupplyResVolumeM3;
```

Comment:
Typo: displacement

This feature of the supply reservoir giving air during emergency seems not so common to me, maybe it could be optional. This looks like we are just duplicating the emergency reservoir.

Reply:
Oops on the typo. As for the supply res in emergency, this is specifically based on equipment that has no emergency reservoir such as 26C brake valves. Note that this section of code won't even run if there is an emergency reservoir.

Reply:
I was thinking in systems with a supply reservoir for the relay valve, but without this emergency feature. I don't see an obvious way to disable this while keeping the supply reservoir for other uses

Reply:
Next version will only enable this if the max triple valve pressure is higher than the max triple valve service pressure (as is the case on the rolling stock the feature is based on). This should disable the feature on all systems where the emergency pressure is not higher than the service pressure.